apps/frontend/src/libs/components/courses/courses.tsx

```diff
@@ -11,22 +12,33 @@ import { Course } from "../course/course.js";
 import styles from "./styles.module.css";
 
 type Properties = {
+	commonCourses?: number[];
```

Comment:
```suggestion
	commonCoursesIds?: number[];
```
check and change it everywhere